vars/buildWDKAutoSwitch.groovy

```diff
@@ -157,7 +170,7 @@ def call(Map config = [unityVersions:[]]) {
                     cleanWs()
                   }
 
-                  ["check Unity-${version}" : helper.transformIntoCheckStep("macos", environment, null, testConfig, checkStep, finalizeStep, true)]
+                  ["check Unity-${version}" : helper.transformIntoCheckStep("macos", environment, null, testConfig, checkStep, finalizeStep, true, catchStep)]
```

Comment:
See my comment about the overload. It could make the invocation look nicer
```suggestion
                  ["check Unity-${version}" : helper.transformIntoCheckStep("macos", environment, null, testConfig, checkStep, catchStep, finalizeStep, true)]
```